### ISSUE TITLE
Ripme 1.7.55 GDPR and SSL on deviantart

### ISSUE THREAD

**Issue by uthr** [RipMeApp/ripme#751]
* Ripme version: 1.7.55
* Java version: java version "10.0.1" 2018-04-17
Java(TM) SE Runtime Environment 18.3 (build 10.0.1+10)
Java HotSpot(TM) 64-Bit Server VM 18.3 (build 10.0.1+10, mixed mode)
* Operating system: windows 10 1709 x64
* Exact URL you were trying to rip when the problem occurred: https://www.deviantart.com/gregchapin/gallery/
* Please include any additional information about how to reproduce the problem:
Expected URL format http://username.deviantart.com/[gallery/####], got: https://www.deviantart.com/gregchapin/gallery/

## Expected Behavior
Downloading all high res images into the \rip folder.

## Actual Behavior
Deviantart has according to the GDPR now SSL. RipMe does not support the new URL structure (yet ;)

##He is an awesome artist, btw
##Thank you for your great work building this piece of software!

**Comment by uthr** [comment]
Yes, exactly! No one noticed :) I was too busy and had no time to watch pictures ;)
I can't even tell, when they switched to `https`...

**Comment by Hrxn** [comment]
They did not made the switch at once, but it was in August 2017 I think..
`https://danlev.deviantart.com/journal/DeviantArt-Is-Switching-To-HTTPS-697996906`

By the way, they also made a change with regard to URL patterns (From `https://<name>.deviantart.com/...` to `https://www.deviantart.com/<name>/..`), but this has been only very recently as it seems.

**Comment by cyian-1756** [comment]
> Deviantart has according to the GDPR now SSL. RipMe does not support the new URL structure (yet ;)

I'm already on fixing this. The reason it's taking a while is because DA also changed how getting the next page of an album works so I need to add support for the DA api to ripme
